developer_setup.md

```diff
@@ -29,6 +29,14 @@ python -m venv ./venv
 pip install .[dev,test]
 ```
 
+Please be informed to use the command below as the square bracket might not be recognized as a pattern.
+
+```shell
+pip install .\[dev,test\]
+
+pip install '.[dev,test]'
```

Comment:
The second one (single quotes) should always work -- just have that without square brackets?

Reply:
Could you please come clearer on your question?

Reply:
There are two pip install commands. Do we need both?

Reply:
Yes, both of them work but I had to remove one. So I think we are good.